package.json

```
@@ -6,9 +6,13 @@
   "directories": {
     "test": "tests"
   },
+  "engines": {
+    "node": "^14.16.1",
+    "npm": "^7.10.0"
```

These seem like really high dependency versions. What is it about these specific versions that are required to make the build work? What would the lowest versions of node and npm be that would still work?

---

> These seem like really high dependency versions. What is it about these specific versions that are required to make the build work? What would the lowest versions of node and npm be that would still work?

These are the latest versions of the LTS node release & the accompanying `latest` tag of npm. While there's nothing tying it to these particular versions functionality-wise, the engine is only developer-facing and it seems reasonable to enforce compatibility with the current point releases.

---

Since it's the latest LTS and latest (minor) release, could it be:

```suggestion
    "node": "^14",
    "npm": "^7"
```

...instead? That gives a bit of wiggle room for contributors to start contributing and building, even if they aren't on the latest minor / patch version.

Right now, the latest npm is 7.11.1.